getting-started/configuring.md

```diff
@@ -1,13 +1,78 @@
+can be limiting when using larger servers with more resources (CPU, memory,
+disk, etc). Adjusting these settings, either
+[automatically with our tool `timescaledb-tune`][tstune] or manually editing
+your machine's `postgresql.conf` can improve performance.
+
+>:TIP: You can determine the location of `postgresql.conf` by running
+`SHOW config_file;` from your PostgreSQL client (e.g., `psql`).
+
+## Recommended: `timescaledb-tune` [](ts-tune)
+
+To streamline the configuration process, we've created a tool called
+[`timescaledb-tune`][tstune] that handles setting the most common settings to
+good values based on your system, accounting for memory, CPU, and PostgreSQL
+version. `timescaledb-tune` is packaged along with our binary releases as
+a dependency, so if you installed one of our binary releases (including Docker)
+you should have access to the tool. Alternatively, with a standard Go
+environment, you can also `go get` the repository to install it.
+
+`timescaledb-tune` reads your system's `postgresql.conf` file and offers
+interactive suggestions for updating your settings:
+```text
+Using postgresql.conf at this path:
+/usr/local/var/postgres/postgresql.conf
+
+Is this correct? [(y)es/(n)o]: y
+Writing backup to:
+/var/folders/cr/zpgdkv194vz1g5smxl_5tggm0000gn/T/timescaledb_tune.backup201901071520
+
+shared_preload_libraries needs to be updated
+Current:
+#shared_preload_libraries = 'timescaledb'
+Recommended:
+shared_preload_libraries = 'timescaledb'
+Is this okay? [(y)es/(n)o]: y
+success: shared_preload_libraries will be updated
+
+Tune memory/parallelism/WAL and other settings? [(y)es/(n)o]: y
+Recommendations based on 8.00 GB of available memory and 4 CPUs for PostgreSQL 11
+
+Memory settings recommendations
+Current:
+shared_buffers = 128MB
+#effective_cache_size = 4GB
+#maintenance_work_mem = 64MB
+#work_mem = 4MB
+Recommended:
+shared_buffers = 2GB
+effective_cache_size = 6GB
+maintenance_work_mem = 1GB
+work_mem = 26214kB
+Is this okay? [(y)es/(s)kip/(q)uit]:
+```
+
+These changes are then written to your `postgresql.conf` and will take effect
+on the next (re)start. If you are starting on fresh instance and don't feel
+the need to approve each group of changes, you can also automatically accept
+and append the suggestions to the end of your `postgresql.conf` like so:
+```bash
+$ timescaledb-tune --quiet --yes --dry-run >> /path/to/postgresql.conf
+```
+
+## Further configuration / manual tuning [](further-config)
+
+If you prefer to tune the settings yourself, or are curious about the
+suggestions that `timescaledb-tune` comes up with, we elaborate on them
+here. Additionally, `timescaledb-tune` does not cover all settings you
+may need to adjust; those are covered below.
 
->:TIP: You can determine the location of `postgresql.conf` by running `SHOW config_file;`
-from within psql.
+### Memory settings [](memory)
 
-## Memory settings [](memory)
+>:TIP: All of these settings are handled by `timescaledb-tune`.
```

Comment:
Are the memory settings the only ones handled by `timescaledb-tune`?    Given that the tip is only here, that's what is suggested.  Please confirm?

Reply:
timescaledb-tune handles some settings in some of the other settings; memory is the only one where it handles all the settings.

Wasn't sure it was worth making tips for each one saying "tune handles this this and this but not this and this". Not sure if I should keep this tip or not either.